components/net/bluetooth_thread.rs

```diff
@@ -790,21 +851,23 @@ impl BluetoothManager {
         let mut characteristics_vec = vec!();
         for characteristic in characteristics {
             if let Ok(uuid) = characteristic.get_uuid() {
-                let properties = self.get_characteristic_properties(&characteristic);
-                characteristics_vec.push(
-                                BluetoothCharacteristicMsg {
-                                    uuid: uuid,
-                                    instance_id: characteristic.get_id(),
-                                    broadcast: properties.contains(BROADCAST),
-                                    read: properties.contains(READ),
-                                    write_without_response: properties.contains(WRITE_WITHOUT_RESPONSE),
-                                    write: properties.contains(WRITE),
-                                    notify: properties.contains(NOTIFY),
-                                    indicate: properties.contains(INDICATE),
-                                    authenticated_signed_writes: properties.contains(AUTHENTICATED_SIGNED_WRITES),
-                                    reliable_write: properties.contains(RELIABLE_WRITE),
-                                    writable_auxiliaries: properties.contains(WRITABLE_AUXILIARIES),
-                                });
+                if !uuid_is_blacklisted(&uuid, Blacklist::All) {
+                    let properties = self.get_characteristic_properties(&characteristic);
+                    characteristics_vec.push(
+                                    BluetoothCharacteristicMsg {
```

Comment:
Maybe because the lines are too long otherwise. We should fix this in another patch.